backend/controller/autocomplete.py

```diff
@@ -14,6 +16,23 @@ class SuggestionResponse(BaseModel):
     suggestions: List[str]
 
 
-@router.get("/query/autocomplete")
+@router.get("/models/{model_id}/faq-qa")
```

Comment:
@wuppersaver  I think it's fine to leave the existing endpint (autocomplete) instead of overwriting the existing one?

Reply:
I wasn't sure, if there was going to be a special folder for this or not and we talked about the possibility of storing searches in the folder i put in. If the old one is fine, im godd with it :)